api/manifest.go

```diff
@@ -56,6 +56,7 @@ type NaisManifest struct {
 	Resources       ResourceRequirements
 	FasitResources  FasitResources `yaml:"fasitResources"`
 	LeaderElection  bool           `yaml:"leaderElection"`
+	Redis           bool           `yaml:"redis"`
```

Comment:
Should we have something like:

services:
  - Redis 
  - Postgres

Reply:
Maybe wait till we have more services to expose?

Reply:
Sure.

Reply:
Then the change wont be backwards compatible? People will have to migrate from `Redis` to the `Services`

Reply:
That is a really good point! 
But, is `services` really the best name? Aren't we talking about `databases`? As `leaderElection`, `prometheus` both also are services that we offer.

Reply:
Services should be a list of services, with each list item having a service type. It's not unheard of that an application depends on multiple services of the same type.
I think `services` is a good name. For example, Kafka is a service, but not a database.

Reply:
I think we need a wider discussion of this, as we need to define what is a service, and what is nais-config. So I will not priority that now.

Reply:
This is more or less the standard here: https://github.com/openservicebrokerapi/servicebroker/blob/v2.12/spec.md

Service: A managed software offering that can be used by an Application. Typically, Services will expose some API that can be invoked to perform some action. However, there can also be non-interactive Services that can perform the desired actions without direct prompting from the Application.
I think Service fits well

Reply:
That's not exactly what I meant. I think we need a discussion if we really want it to be stacked like this, or just continue with what we have now. And, if we want to change it, what do we define as a service and what should stay on the top.

So, I'm still voting for not doing anything with it in this PR.

Reply:
I think Redis and LeaderElection belongs under services. 
In my head those are things outside the normal nais-use case.
Prometheus is config.